subsquid/src/processor.ts

```diff
@@ -67,18 +60,18 @@ processor.addEventHandler("BondedFinance.NewOffer", processNewOfferEvent);
 
 processor.addEventHandler("BondedFinance.NewBond", processNewBondEvent);
 
-processor.addEventHandler(
-  "BondedFinance.OfferCancelled",
-  processOfferCancelledEvent
-);
+processor.addEventHandler("BondedFinance.OfferCancelled", processOfferCancelledEvent);
 
-processor.addEventHandler(
-  "Vesting.VestingScheduleAdded",
-  processVestingScheduleAddedEvent
-);
+processor.addEventHandler("Vesting.VestingScheduleAdded", processVestingScheduleAddedEvent);
 
 processor.addEventHandler("Vesting.Claimed", processVestingClaimedEvent);
 
+processor.addEventHandler("*", async ctx => {
+  // console.log(ctx.event.name);
+  // console.log(ctx.event.extrinsic);
+  // console.log(ctx.event.extrinsic?.success);
+});
```

Comment:
Looks like this is doing nothing. It can be removed.

Reply:
Done ✅